brand/schema.py

```diff
@@ -255,11 +255,16 @@ class Query(graphene.ObjectType):
     commentary = relay.Node.Field(Commentary)
     commentaries = DjangoFilterConnectionField(Commentary)
 
-    brand = graphene.Field(Brand, tag=graphene.Argument(graphene.String, required=True))
+    brand_by_tag = graphene.Field(Brand, tag=graphene.Argument(graphene.String, required=True))
```

Comment:
This is a breaking change -- anyone relying on the old endpoint will need to be updated.  Unless you know where this is called and have corresponding updates in those repos I would leave this as is.

Reply:
Yeah makes sense, I forgot that this would be called in other places and possibly affect them too.